pages/models.py

```diff
@@ -615,6 +620,29 @@ class Adopters(Page):
     parent_page_types = ['pages.HomePage']
 
 
+class Organizations(models.Model):
+    salesforce_id = models.CharField(max_length=255)
```

Comment:
you should probably make this non editable:
`salesforce_id = models.CharField(max_length=255, editable=False)`